src/services/campaignConfig/campaignConfig.test.ts

```diff
@@ -14,24 +15,97 @@ const mockValidResponse = {
     minAppBinaryVersion: "3.0.0",
     minAppBuildVersion: 0,
     campaignName: "Test campaign",
+    transactionGrouping: true,
     flowType: "DEFAULT",
-    transactionGrouping: true
-  }
+    id: {
+      type: "STRING",
+      scannerType: "CODE_39",
+      validation: "NRIC"
+    }
+  },
+  policies: [
+    {
+      category: "toilet-paper",
+      name: "🧻 Toilet Paper",
+      description: "1 ply / 2 ply / 3 ply",
+      order: 1,
+      quantity: {
+        period: 7,
+        limit: 2,
+        unit: {
+          type: "POSTFIX",
+          label: " pack(s)"
+        }
+      }
+    },
+    {
+      category: "chocolate",
+      name: "🍫 Chocolate",
+      description: "Dark / White / Assorted",
+      order: 2,
+      quantity: {
+        period: 14,
+        limit: 30,
+        step: 5,
+        unit: {
+          type: "PREFIX",
+          label: "$"
+        }
+      }
+    }
+  ]
 };
 
 const mockValidResponseNewFeature = {
   features: {
+    newFeature: true,
```

Comment:
is this just a dummy value to show that there's a change in the response HAHA

Reply:
yea, to indicate that if there's a new property  not defined in the current `policies` or `features`,  nothing breaks, it'll be safely ignored

Reply:
ahh, okok! ^^